app/mainwindow2.h

```diff
@@ -91,6 +91,8 @@ public slots:
 
     void importMovie();
 
+    void lockWidgets( bool shouldLock );
```

Comment:
You're right, it probably should. You'll notice that most of the other functions in this class have a similar issue, so it will require a bit more refactoring to fully correct the issue.